composer.json

```diff
@@ -24,7 +24,9 @@
         "issues": "https://github.com/RusticiSoftware/TinCanPHP/issues"
     },
     "require": {
-        "php": "~5.4"
+        "php": "~5.4",
+        "ext-openssl": "*",
+        "namshi/jose": "~3.0"
```

Comment:
Just noting that right now, this does not work. Presumably it will work when the PR into namshi/jose is merged.